styles/stepper.css

```diff
@@ -104,3 +104,36 @@
         padding-left: 1rem;
     }
 }
+
+.circle-progress-indicator {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    top: 15px;
+    width: 100%;
+    height: 100%;
+    margin: 0 auto;
+    user-select: none;
+    position: absolute;
+    width: 59px;
+    height: 30px;
+
+    font-family: Lato;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 30px;
+    color: #259F46;
+    letter-spacing: 0.1px;
+}
+
+.RCP {
+    width: 50px;
+    margin-left: 0px;
+
+}
+
+svg {
```

Comment:
Targeting all `<svg>` attributes is not a good practice. Tailwind prefers an inline svgs with a `class=` attribute to modify svgs

Reply:
done